src/Traversals/dfs.jl

```diff
@@ -58,14 +73,14 @@ end
     s.w = 0
     return true
 end
-@inline function visitfn!(s::TopoSortState{T}, u, v) where T 
+@inline function visitfn!(s::TopoSortState{T}, u, v) where T
```

Comment:
is `@inline` needed in all the following functions?

Reply:
see above